src/sysv/systemd/sssd-ifp.service.in

```diff
@@ -15,3 +15,4 @@ CapabilityBoundingSet= @additional_caps@
 Restart=on-failure
 User=@SSSD_USER@
 Group=@SSSD_USER@
+PermissionsStartOnly=true
```

Comment:
Hi,

wouldn't it be better to change the other unit files and drop `PermissionsStartOnly` or are you planning this for an upcoming major release?

bye,
Sumit

Reply:
One patch fixes 'ifp' service description, another patch replaces deprecated option with more modern syntax.
I can squash them, but why?